src/box/applier.cc

```diff
@@ -2890,6 +2911,7 @@ void
 applier_resume_to_state(struct applier *applier, enum applier_state state,
 			double timeout)
 {
+	assert(state != APPLIER_STOPPED && state != APPLIER_OFF);
```

Comment:
What's wrong with these states? AFAIU this function should work just fine for them too.

Reply:
Dropped. This is remnant of the previous version of the patch where diag was stealed in `applier_wait_for_state`.